test-ai-review.py

```
@@ -0,0 +1,172 @@
+        return self.users.get(user_id,None)
+
+# 🚨 크리티컬 이슈: 예외 처리 없음
+def download_file(url, filename):
+    response = requests.get(url)
+    with open(filename, 'wb') as file:
+        file.write(response.content)
+    return filename
+
+# ⚠️ 리소스 관리 이슈: 파일 핸들 누수 위험
+def read_config():
+    file = open('config.txt', 'r')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)